src/Components/Products/Products.css

```diff
@@ -35,24 +35,69 @@
     border-radius: 4px;   
 }
 
-@media screen and (max-width: 767px) {
+@media (max-width: 1024px) {
     .products__col {
         display: flex;
         flex-direction: column;
         min-height: 230px;
-        width: 48%;
+        width: 30.8%;
```

Comment:
Replace with **width: 33.333%**
adjust css rules logic to fit width

Reply:
Fixed.